app/mailers/record_action_mailer.rb

```diff
@@ -42,6 +42,19 @@ def transfer_request(transfer_request_notification)
     mail(to: @transfer_request_notification&.transitioned_to&.email, subject: @transfer_request_notification.subject)
   end
 
+  def alert_notify(alert_notification)
+    @alert_notification = alert_notification
+    @user = @alert_notification.user
```

Comment:
@user, @locale_email and @record are not need it in the view you can use @alert_notification